pkg/operator/starter.go

```diff
@@ -192,7 +192,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		deploymentHooks = []dc.DeploymentHookFunc{
 			hyperShiftReplaceNamespaceHook(controlPlaneNamespace),
 			hyperShiftRemoveNodeSelector(),
-			hyperShiftAddKubeConfigVolume("admin-kubeconfig"), // TODO: use dedicated secret for Snapshots
+			hyperShiftAddKubeConfigVolume("service-network-admin-kubeconfig"), // TODO: use dedicated secret for Snapshots
```

Comment:
We we need to update the AWS operator as well? https://github.com/openshift/aws-ebs-csi-driver-operator/blob/9407ad79fbcae7231fb94d4dd69f264af8378129/pkg/operator/starter.go#L533

Reply:
Probably so.  Not sure why we haven't seen that pod crashing as well.